packages/amplify_authenticator/lib/src/enums/verified_attributes.dart

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+/// Uses snake case to conform with cognito service
```

Comment:
I had this originally, but changed it because the two enums are used for very different purposes. I can see the value in consolidating though - I'll change these back.

Reply:
I think another option would be to make one enum `CognitoUserAttribute` to replace `UsernameAttribute` and `VerifiedAttributes`. It might even replace the need for `SignUpField`. `UsernameAttribute`, `VerifiedAttributes`, and `SignUpField` are all just subsets of a more comprehensive enum that includes all user attributes (`CognitoUserAttribute`). I see the potential advantage of having a more specific enums, but right now it seems like they are used somewhat interchangeably.

I might look into the option of combining them.